### ISSUE TITLE
[cli] run.sh designer reports "integer expression expected"

### ISSUE THREAD

**Issue by adangel** [pmd/pmd#4118]
**Affects PMD Version:** 6.49.0

**Description:**

Java versions now may have a 4-part version number. That seems to confuse the shell script to determine the java version:

```shell
$ java -version
openjdk version "11.0.16.1" 2022-08-12
OpenJDK Runtime Environment Temurin-11.0.16.1+1 (build 11.0.16.1+1)
OpenJDK 64-Bit Server VM Temurin-11.0.16.1+1 (build 11.0.16.1+1, mixed mode)
$ bin/run.sh designer
bin/run.sh: line 150: [: 11.016: integer expression expected
bin/run.sh: line 110: [: 11.016: integer expression expected
bin/run.sh: line 114: [: 11.016: integer expression expected
PMD Rule Designer (v 6.49.0) initializing... 
done in 4012ms.
Run with --verbose parameter to enable error output.
```

The designer works, but the warnings sound like there is a bug.
